backend/index.js

```diff
@@ -0,0 +1,99 @@
+const express = require("express");
+const app = express();
+const orm = require("sequelize");
+const database = new orm.Sequelize({
+  dialect: "sqlite",
+  storage: "./data/catsDatabase.sqlite",
+});
+
+const Cat = database.define("Cat", {

```

Comment:
Should move in a separe file 
Example in a "model" foldel

Mayne use repository pattern (https://4markdown.com/understanding-repository-pattern-in-nodejs-and-typescript/)